indra/tools/reading/read_db_aws.py

```diff
@@ -91,7 +98,25 @@
 
     # Read everything ========================================
     outputs = produce_readings(id_dict, readers, verbose=True,
-                               read_mode=args.mode)
+                               read_mode=args.mode,
+                               force_fulltext=args.force_fulltext)
+
+    # Preserve the sparser logs
+    contents = os.listdir('.')
+    sparser_logs = [fname for fname in contents
+                    if fname.startswith('sparser') and fname.endswith('log')]
```

Comment:
You could replace this with just glob.glob('sparser*.log') for simplicity.

Reply:
Oh, true. I almost never use glob. Is it a standard part of Python, or does it need special installation?